kubeshop/botkube
# Update JSON schemas for all plugins

## Description

Changes proposed in this pull request:

- Update JSON schemas for all plugins
   - Add password format for credentials
   - Ensure `log` is exposed everywhere
   - Ensure `additionalProperties` are enabled for plugins with more complex data
- Unify JSON schema definition across all plugins

## Testing

- rebuild your local plugins  `make build-plugins-single`
- run your local plugin index server `go run test/helpers/plugin_server.go`
- use `http://localhost:3010/botkube.yaml` as plugin repository for Botkube Cloud
- Create instance and see the plugin list. Enable all of them and see the YAML editor + UI forms.
   - See that all the propoerties are already filled-in in the yaml editor
   - See that the password fields are properly visible in UI form

![image](https://github.com/kubeshop/botkube/assets/7155799/292b3c2a-d82a-4653-bd9b-0ec553a3ac3d)


## Related issue(s)
